examples/federated-learning/func_lib.py

```diff
@@ -0,0 +1,63 @@
+import tensorflow as tf
+import tf_encrypted as tfe
+
+
+### Example model_fns ###
+
+def default_model_fn(data_owner):
```

Comment:
`model_fn` to me is what constructs the model, as [here in TFF](https://www.tensorflow.org/federated/federated_learning#converters_for_keras)

Reply:
I was referring to how tf.Estimator refers to it: 
>The Estimator object wraps a model which is specified by a model_fn, which, given inputs and a number of other parameters, returns the ops necessary to perform training, evaluation, or predictions.

Happy to change the name though!